apps/eo_web/src/helpers/sentry.ts

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import * as Sentry from "@sentry/react";
+import {
+  createRoutesFromChildren,
+  matchRoutes,
+  useLocation,
+  useNavigationType,
+} from "react-router-dom";
+
+import { env } from "~/env";
+
+
+const sentryDns =
+  window.data.getEnv("VITE_SENTRY_DSN_PUBLIC") ?? env.VITE_SENTRY_DSN_PUBLIC;
+
+if (sentryDns) {
+  Sentry.init({
+    environment: window.data.getEnv("VITE_ENV") ?? env.VITE_APP_ENV,
```

Comment:
What if the VITE_APP_ENV is not set?

Also shouldn't it be like this?:

```suggestion
    environment: window.data.getEnv("VITE_APP_ENV") ?? env.VITE_APP_ENV,
```